website/docs/components/table/advanced-table/partials/code/component-api.md

```diff
@@ -0,0 +1,204 @@
+## Component API
+
+The AdvancedTable component itself is where most of the options will be applied. However, the APIs for the child components are also documented here, in case a custom implementation is desired.
+
+### AdvancedTable
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="<:body>" @type="named block">
+    This is a named block where the content for the AdvancedTable body is rendered.
+    <Doc::ComponentApi as |C|>
+      <C.Property @name="[B].rowIndex" @type="number | string">
+        The value of the index associated with the `@each` loop. Returns a number when there are no nested rows. Returns a string in the form `${parentIndex}.${childIndex}` when there are nested rows.
+      </C.Property>
+      <C.Property @name="[B].sortBy" @type="string">
+        The value of the internal `sortBy` tracked variable.
+      </C.Property>
+      <C.Property @name="[B].sortOrder" @type="string">
+        The value of the internal `sortOrder` tracked variable.
+      </C.Property>
+      <C.Property @name="[B].isExpanded" @type="boolean">
+        The value of the internal `isExpanded` tracked variable from the row if it has nested rows. Otherwise returns `undefined`.
```

Comment:
```suggestion
        Returns the value of the internal `isExpanded` tracked variable from the row if it has nested rows. Otherwise returns `undefined`.
```